omero/training-server/playbook.yml

```diff
@@ -280,7 +292,7 @@
     omero_webtagging_tagsearch_release: "{{ omero_webtagging_tagsearch_release_override | default('3.0.3') }}"
     omero_cli_duplicate_release: "{{ omero_cli_duplicate_release_override | default('0.2.0') }}"
     omero_cli_render_release: "{{ omero_cli_render_release_override | default('0.3.0') }}"
-    os_system_users_password: "{{ os_systems_users_password_override | default('$6$leKi5B1PgSvdA/ec$xbU3CnoSFnYdeZjEjKK5TH8SGATsW746uopssff4edpgyu.cWXGo9A.oK6wH9kIkxLCCNcORGZnnroZPMqGzN/') }}"
+    os_system_users_password: "{{ os_systems_users_password_override | default('$6$sjH0EeRKZdWYc1y8$F79INhPIfwVUfZ/DRrk.rYEetacoxoqxMVSVRavjggro84vpCqcxUhcNs79C5y4RmdLBb/3sA3vRagfNVjMxD.') }}"
```

Comment:
I've discovered there's also https://github.com/openmicroscopy/management_tools/blob/master/ansible/inventory/host_vars/outreach.openmicroscopy.org.yml#L1
Presumably the original intention was that the password in this public file could be well known, and we'd override it with the linked private password. If this causes too much confusion we could delete the private override?

Reply:
I do not hink this is a matter of confusion, rather development of the ecosystem.

Will update the pwd in the MT as well.